custom_completions.go

```diff
@@ -442,7 +442,11 @@ func checkIfFlagCompletion(finalCmd *Command, args []string, lastArg string) (*p
 	if len(lastArg) > 0 && lastArg[0] == '-' {
 		if index := strings.Index(lastArg, "="); index >= 0 {
 			// Flag with an =
-			flagName = strings.TrimLeft(lastArg[:index], "-")
+			if strings.HasPrefix(lastArg[:index], "--") {
+				flagName = lastArg[2:index]
+			} else {
```

Comment:
Could you put a comment to mention the handling the scenario of multiple short-hand flag, maybe with a short example?  I feel that in 2 months, I won't remember why we do this 😄